johncena.arnoldc

```diff
@@ -0,0 +1,5 @@
+IT'S SHOWTIME
+STICK AROUND @NO PROBLEMO
+TALK TO THE HAND "\n"
```

Comment:
i think this should be "" only.

Reply:
No, because that's empty string. \n is the escape character for newline. I don't know about the other implementations but if you look at the js one it uses \n as well

Reply:
I tried on online complier. It was not showing escape character but rather printing /n itself